<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cncli</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEditTerminal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>439</y>
      <width>830</width>
      <height>184</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>200</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>403</width>
      <height>382</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxPosition">
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBoxY_2">
          <property name="title">
           <string>X</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Position (mm)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxXPosition">
                 <property name="decimals">
                  <number>9</number>
                 </property>
                 <property name="maximum">
                  <double>999.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.012500000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Step size (mm)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxXSteps">
                 <property name="decimals">
                  <number>5</number>
                 </property>
                 <property name="minimum">
                  <double>0.012500000000000</double>
                 </property>
                 <property name="maximum">
                  <double>20.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.012500000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxY">
          <property name="title">
           <string>Y</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Position (mm)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxYPosition">
                 <property name="decimals">
                  <number>9</number>
                 </property>
                 <property name="maximum">
                  <double>999.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.012500000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Step size (mm)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxYSteps">
                 <property name="decimals">
                  <number>5</number>
                 </property>
                 <property name="minimum">
                  <double>0.012500000000000</double>
                 </property>
                 <property name="maximum">
                  <double>20.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.012500000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxY_3">
          <property name="title">
           <string>Z</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Position (mm)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxZPosition">
                 <property name="decimals">
                  <number>9</number>
                 </property>
                 <property name="maximum">
                  <double>999.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.012500000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Step size (mm)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxZSteps">
                 <property name="decimals">
                  <number>5</number>
                 </property>
                 <property name="minimum">
                  <double>0.012500000000000</double>
                 </property>
                 <property name="maximum">
                  <double>20.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.012500000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft|Qt::AlignTop">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Tools</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonSendInit">
          <property name="text">
           <string>Send Init Sequence</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonMoveToZero">
          <property name="text">
           <string>Move To Zero</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
